Homework_6_Java_Intro/MyFirstJavaProject/src/Main.java

```diff
@@ -0,0 +1,5 @@
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("Hello my first automation test!");
```

Comment:
🥇 
Well done! (this is not a test yet, but anyway this can be called your first Java program, which is even cooler :)